test/conftest.py

```diff
@@ -469,3 +469,15 @@ async def solr_search(solr_config, app_manager):
     assert result.migrations_run == len(entity_schema.all_migrations)
 
     return solr_config
+
+
+@pytest.hookimpl(tryfirst=True)
+def pytest_runtest_setup(item):
+    mark = item.get_closest_marker(name="myskip")
+    if mark:
+        condition = next(iter(mark.args), True)
+        reason = mark.kwargs.get("reason")
+        item.add_marker(
+            pytest.mark.skipif(not os.getenv("PYTEST_FORCE_RUN_MYSKIPS", False) and condition, reason=reason),
+            append=False,
+        )
```

Comment:
Can you please either remove this full and just use `pytest.mark.skip` or rename the custom marker to something else. Like "external_service". Then by default you will skip all tests that depend on an external service (like openbis) but if you set `PYTEST_FORCE_RUN_EXTERNAL_SERVICE` then the tests will run. "myskip" is a bit too vague.